docs/Permissions/Permissioning.md

```diff
@@ -9,23 +9,23 @@ account permissions enabled, or both.
 
 ## Node Whitelisting 
 
-Node whitelisting is specified by the nodes whitelist in the [`permissions_config.toml`](#permissions-configuration-file) file. 
+Node whitelisting is specified by the nodes whitelist in the [permissions configuration file](#permissions-configuration-file) file. 
 A node with node whitelisting enabled communicates only with nodes in the nodes whitelist. 
```

Comment:
Can we replace"
`A node with node whitelisting enabled communicates only with nodes in the nodes whitelist.`
With
`A node with node whitelisting enabled communicates only with nodes in the whitelist.`
Too many `nodes` made the statement a bit confusing :)

Reply:
How about "With node whitelisting enabled, communication is restricted to only nodes in the whitelist"

Reply:
gets it down to 2 "nodes"

Reply:
Fixed - used Sally's suggestion